docs/src/python-api.md

```diff
@@ -0,0 +1,85 @@
+# Python API
+
+## Cutom Node
+
+The custom node API allow you to integrate `dora` into your application. It allows you to retrieve input and send output in any fashion you want.  
+#### `Node()`
+
+`Node()` initiate a node from environment variables set by `dora-coordinator` 
+
+```python
+from dora import Node()
+
+node = Node()
+```
+
+#### `.next()` or `__next__()` as an iterator
+
+`.next()` gives you the next input that the node has received.
```

Comment:
```suggestion
`.next()` gives you the next input that the node has received. It blocks until the next input becomes available.
```

Reply:
So for python, it will not block. It will send a `None` value if there is no available input in the `tokio::channel`. I will add that to the documentation

Reply:
Are you sure? I think we're doing `blocking_recv` on the channel, which will only return `None` when the channel is closed.

Reply:
Yeah, you're right. I just checked the documentation again, and it in fact, block it until the sender is dropped, this is where it sends None.